src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -120,13 +150,13 @@ describe('HmctsGlobalHeaderComponent', () => {
       href: '',
       active: false
     }];
-    await component.ngOnChanges(changesMock);
+    component.ngOnInit();
     fixture.detectChanges();
-    const findCase = fixture.debugElement.query(By.css('.hmcts-search-toggle__button'));
-    expect(findCase).toBeTruthy();
+    component.isUserCaseManager$.subscribe(result => expect(result).toBe(true));
   });
 
-  it('should not display find case right aligned', async () => {
+  it('should not display find case right aligned', () => {
```

Comment:
@johnbenjamin-hmcts - as per previous comment above on whether the test has asynchronous behaviour.